### PR TITLE
Feat/fetch output submitter info

### PR DESCRIPTION
**Pull request by sh-cha**
`OutputSubmitter` now only needs to register the proposer that exists in bridge info to the keyring instead of writing the keyname in config.

`EnableOutputSubmitter` must be set to true to make OutputSubmitter work.

## Summary by CodeRabbit

- **New Features**
	- Enhanced configuration management for bridge information and keyring handling across various components.
	- Introduced `Bech32Prefix` field in node configuration for improved chain prefix management.

- **Bug Fixes**
	- Streamlined initialization processes to ensure compatibility with new parameter types, enhancing overall stability.

- **Documentation**
	- Updated method signatures and configuration structures for clarity and improved usability.

- **Refactor**
	- Simplified method signatures by removing unnecessary parameters, improving code readability and maintainability.

- **Chores**
	- Removed deprecated import statements and adjusted dependencies accordingly.

### CODE REVIEW

File: node/broadcaster/types/config.go [hunk 1]
```diff
@@ -37,6 +37,9 @@ type BroadcasterConfig struct {
 	// PendingTxToProcessedMsgs is the function to convert pending tx to processed messages.
 	PendingTxToProcessedMsgs PendingTxToProcessedMsgsFn
 
+	// HomePath is the path to the keyring.
+	HomePath string
+
 	// KeyringConfig is the keyring configuration.
 	KeyringConfig KeyringConfig
```
Comment (beer-1) [inline node/broadcaster/types/config.go]: maybe need to delete it?

**Review by beer-1 — CHANGES_REQUESTED** [review]
Left one comment

File: node/broadcaster/types/config.go (hunk 1 above)
Comment (sh-cha) [reply node/broadcaster/types/config.go:44]: we need it to be used to get keyring base

**Review by beer-1 — APPROVED** [review]
LGTM

**Pull request merged**